### ISSUE TITLE
 ID-MRF loss

### ISSUE THREAD

**Issue by leishuangshuang** [shepnerd/inpainting_gmcnn#31]
I did not modify any parameters during my training. Why did the ID-MRF loss increase during the second phase of training?

**Comment by shepnerd** [comment]
May I ask about your some experimental settings, e.g., the used dataset?

**Comment by leishuangshuang** [comment]
dataset:paris_streetview,batch_size:8,pretrain_network:0.
The downloaded pre-training model is a model that is set when pretrain_network is set to 1. The other parameters are default parameters.




------------------ 原始邮件 ------------------
发件人: "Yi Wang"<notifications@github.com>;
发送时间: 2019年9月23日(星期一) 下午3:33
收件人: "shepnerd/inpainting_gmcnn"<inpainting_gmcnn@noreply.github.com>;
抄送: "亲耐嘚?苹果"<1046044085@qq.com>;"Author"<author@noreply.github.com>;
主题: Re: [shepnerd/inpainting_gmcnn]  ID-MRF loss (#31)




May I ask about your some experimental settings, e.g., the used dataset?

**Comment by shepnerd** [comment]
Do you mean you fine-tine our given pre-trained model and find the id-mrf loss increase during the second training phase?

**Comment by leishuangshuang** [comment]
The pre-training model is my own training, and its reconstruction loss has been declining, but its ID-MRF loss has been rising while using my own pre-training model during the second training phase.





------------------ 原始邮件 ------------------
发件人: "Yi Wang"<notifications@github.com>;
发送时间: 2019年9月23日(星期一) 下午5:07
收件人: "shepnerd/inpainting_gmcnn"<inpainting_gmcnn@noreply.github.com>;
抄送: "亲耐嘚?苹果"<1046044085@qq.com>;"Author"<author@noreply.github.com>;
主题: Re: [shepnerd/inpainting_gmcnn]  ID-MRF loss (#31)




Do you mean you fine-tine our given pre-trained model and find the id-mrf loss increase during the second training phase?

**Comment by shepnerd** [comment]
Do you finish the first training phase? I mean the reconstruction loss barely decreases from the trend. I just doubt that the optimizing procedure is still dominated by the reconstruction loss. If so, it is possible that id-mrf loss rises in the optimization.

**Comment by leishuangshuang** [comment]
Ok, let me see first, thank you very much.


------------------ 原始邮件 ------------------
发件人: "Yi Wang"<notifications@github.com>;
发送时间: 2019年9月24日(星期二) 下午2:12
收件人: "shepnerd/inpainting_gmcnn"<inpainting_gmcnn@noreply.github.com>;
抄送: "亲耐嘚?苹果"<1046044085@qq.com>;"Author"<author@noreply.github.com>;
主题: Re: [shepnerd/inpainting_gmcnn]  ID-MRF loss (#31)




Do you finish the first training phase? I mean the reconstruction loss barely decreases from the trend. I just doubt that the optimizing procedure is still dominated by the reconstruction loss. If so, it is possible that id-mrf loss rises in the optimization.
